contracts/SimpleEscrowEvents/simple_escrow_events.sol

```diff
@@ -1,16 +1,16 @@
 pragma solidity ^0.4.0;
 contract SimpleEscrowEvents {
 
-    event Payment(string indexed contractAddress, uint amount);
+    event Payment(string tokenContractAddress, uint tokenAmount, string destinationStellarAddress);
```

Comment:
removed indexes on the event since we need the pre-image of the destinationStellarAddress.

since we are hardcoding the tokenContractAddress to a limited set we were previously inferring the data based on the hash but for now to keep things simple I've removed that from the index too so it is simpler to get the information from the event directly